### PR TITLE
Add 2.2 find person requirements and samples

### PR DESCRIPTION
**Pull request by sakulich**

### CODE REVIEW

File: Find Person/2.2.md [hunk 1]
```diff
@@ -0,0 +1,302 @@
...
+| api_key | API key. REQUIRED | |
+| name | Person name as a single parameter | Jane Smith |
+| first_name | Person’s first name | Jane |
+| middle_name | Could be name or initial | R |
+| last_name | Person’s surname | Smith |
+| street_line_1 | Number and street name | 2808 Nero Blvd |
+| street_line_2 | Apartment or other additional address information | Apt 265 |
+| city | City name | Seattle |
+| postal_code | 5 or 9 digit US or 6 digit Canadian zip code | 92019 or S3D 3F3 |
+| state_code | 2 character state code | WA |
```
Comment (sakulich) [inline Find Person/2.2.md]: Dont understand what is wrong here.

File: Find Person/2.2.md [hunk 2]
```diff
@@ -0,0 +1,302 @@
...
+
+
+** Locations array**
+
+| Field     | Type | Description |
+| ------- | ---- | ---- |
+| id | Object | key, type, uuid, durability |
+| type | String | Type of the address |
+| valid_for | Object | Start and End day, month, year when person was associated with address |
+| Full address |  | To fully describe the location we return following strings and boolean values: legal_entities_at, city, postal_code, zip4, state_code, country_code, is_receiving_mail, not_receiving_mail_reason, usage, delivery_point, address_type, is_deliverable, standard_address_line1, standard_address_line2, is_historical  |
```
Comment (sakulich) [inline Find Person/2.2.md]: I dont think we should create one more table explaining the attributes.

File: Find Person/2.2.md [hunk 3]
```diff
@@ -0,0 +1,302 @@
...
+| use_historical | Search person using historical address match | true |
+| use_metro | Expand search to metro/suburban area of a given city/state or zip | true |
+
+
+## Response
+A Find Person response is formatted as follows:
+
+| Field     | Type | Description |
+| ------- | ---- | ---- |
+| id | object | key, type |
```
Comment (sakulich) [inline Find Person/2.2.md]: fixed one and it also depends on id.

File: Find Person/2.2.md (hunk 1 above)
Comment (csgero) [inline Find Person/2.2.md]: Just wanted to be more precise, as there are multiple possible 2 character state codes.

File: Find Person/2.2.md [hunk 4]
```diff
@@ -0,0 +1,302 @@
...
+
+
+** Locations array**
+
+| Field     | Type | Description |
+| ------- | ---- | ---- |
+| id | object | key, type, uuid, durability |
+| type | string | Type of the address |
+| valid_for | object | Start and End day, month, year when person was associated with address |
+| Full address |  | To fully describe the location we return following strings and boolean values: legal_entities_at, city, postal_code, zip4, state_code, country_code, is_receiving_mail, not_receiving_mail_reason, usage, delivery_point, address_type, is_deliverable, standard_address_line1, standard_address_line2, is_historical  |
```
Comment (csgero) [inline Find Person/2.2.md]: Why are all these fields crammed into this row? Shouldn't these be separate rows? Also I'm quite sure that legal_entities_at is neither string nor boolean.

File: Find Person/2.2.md [hunk 5]
```diff
@@ -0,0 +1,302 @@
...
+| ------- | ---- | ---- |
+| id | object | key, type |
+| names | array | Name and alias data for this person. The names could include any of the following optional fields: salutation, first_name, middle_name, last_name, suffix |
+| age_range | object | start, end  |
+| gender | string | Either “Male” or “Female” |
+| locations | array | Current and prior physical addresses for this entity |
+| phones | array | Current and historical land-line phones for this person. |
+
+
+** Locations array**
```
Comment (raszi) [inline Find Person/2.2.md]: Unnecessary spaces after the first double star.

**Pull request merged**